lms/resources/_js_config/__init__.py

```diff
@@ -444,21 +444,14 @@ def _get_product_info(self):
         }
 
         if self._request.product.settings.groups_enabled:
-            # This should be abstracted as "lms_course_id" or similar in either
-            # LTIParams or as part of product.
-            api_course_id = self._request.lti_params.get(
-                "custom_canvas_course_id"
-            ) or self._request.lti_params.get("context_id")
-
             product_info["api"]["listGroupSets"] = {
                 "authUrl": self._request.route_url(product.route.oauth2_authorize),
-                "path": self._request.route_path(
-                    "api.courses.group_sets.list", course_id=api_course_id
-                ),
+                "path": self._request.route_path("api.courses.group_sets.list"),
                 "data": {
                     "lms": {
                         "product": self._request.product.family,
-                    }
+                    },
+                    "context_id": self._request.lti_params["context_id"],
```

Comment:
Keep it in the URL for now.

We probably should build a common set of parameters for all/most of our internal API endpoints, we might not need `context_id` everywhere but why not just send it along product, resource_link_id and some others. With a documented schema etc it will be much clearer what that we have available.

I'll leave that for other PR in any case.